src/Symfony/Component/Notifier/Bridge/Ntfy/NtfyTransport.php

```diff
@@ -41,6 +42,13 @@ public function getTopic(): string
         return $this->topic;
     }
 
+    public function setToken(?string $token): self
```

Comment:
```
public function(#[\SensitiveParameter] ?string $token): self
```

(Probably useless as you may remove the method :) )

Reply:
Is `#[\SensitiveParameter]` already in the 6.4 branch? `setPassword` has it in 7.0 but not in 6.4.

Reply:
Sorry forgot you targetted 6.4. 

But as i'm not sure it will be considered a bug ... you may have to target 7.1 (let's wait to see what admins say)